website/static/js/fangorn.js

```diff
@@ -2230,6 +2245,25 @@ tbOptions = {
     uploads : true,         // Turns dropzone on/off.
     columnTitles : _fangornColumnTitles,
     resolveRows : _fangornResolveRows,
+    // Move obj.data.attributes contents to obj.data , for WB v1 API
+    lazyLoadPreprocess: function(obj) {
+        if (!$.isArray(obj.data)) {
+            $.each( obj.data.attributes, function( key, value ) {
+                    obj.data[key] = value;
+            });
+            delete obj.data.attributes;
+            return obj;
+        }
+        /* jshint ignore:start */
+        for (var i = 0; i < obj.data.length; i++) {
+            $.each( obj.data[i].attributes, function( key, value ) {
```

Comment:
I think you could iterate over `obj.data` with `$.each`, and do `$.extend(entity, entity.attributes)` to copy the properties up a level.